ui/components/Header.js

```diff
@@ -423,7 +423,7 @@ function K8sContextMenu({
                             handlePing={() => ping(ctx.name, ctx.server, ctx.connection_id)}
                             iconSrc={
                               connectionMetadataState
-                                ? connectionMetadataState[CONNECTION_KINDS.KUBERNETES]?.icon
+                                ? `/${connectionMetadataState[CONNECTION_KINDS.KUBERNETES]?.icon}`
```

Comment:
You sure it is required? What is breaking otherwise?

Reply:
@MUzairS15 depending upon what page you're on in Meshery UI, the relative URL doesn't reach the SVG, apparently. This absolute reference does, though.